### DISCUSSION TITLE
Spectorgram settings

### DISCUSSION THREAD

**Discussion by huminator1978** [mcguirepr89/BirdNET-Pi#1264] [Q&A]
I am new to birdnet pi and the coding but I am looking for explanation settings Spectorgram. So if these are noob questions I am already sorry :) but I can't find clear answers.

1: What is the best % for gain?
2: What is compression and do you and do turn this on or off?
3: What is freq shift and do you and do turn this on or off?

![Spectogram](https://github.com/user-attachments/assets/56e1875e-0f54-4b1c-9d07-c482ca6e4c79)

**Comment by stevedee** [comment]
The settings on the Spectrogram page do not affect analysis. They provide a useful way of checking audio and maybe comparing one setup to another (e.g. you could take screenshots to compare (say) noise with one configuration to another). 
None of the settings on the Spectrogram page are persistent, they are reset next time you open this page.
Compression raises the volume of low level sound and Freq Shift changes the frequency of all sounds, which may help you hear frequencies which are just outside your personal hearing/frequency range.

**Reply by huminator1978** [reply]
Thanks
